extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -60,16 +64,35 @@ export const DOUBLECLICK_A4A_INTERNAL_EXPERIMENT_BRANCHES = {
   experiment: '117152681',
 };
 
+/** @const {!../../../ads/google/a4a/traffic-experiments.ExperimentInfo} */
+export const DOUBLECLICK_A4A_BETA_BRANCHES = {
+  control: '2077830',
+  experiment: '2077831',
+};
+
 /**
  * @param {!Window} win
  * @param {!Element} element
  * @returns {boolean}
  */
 export function doubleclickIsA4AEnabled(win, element) {
-  // Ensure not within remote.html iframe.
-  return !win.document.querySelector('meta[name=amp-3p-iframe-src]') &&
-      googleAdsIsA4AEnabled(
-        win, element, DOUBLECLICK_A4A_EXPERIMENT_NAME,
-        DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES,
-        DOUBLECLICK_A4A_INTERNAL_EXPERIMENT_BRANCHES);
+  const usesRemoteHTML =
+      !!win.document.querySelector('meta[name=amp-3p-iframe-src]');
+  const a4aRequested = !!element.getAttribute(
+      'data-use-experimental-a4a-implementation');
```

Comment:
This was @michaelkleber 's suggested name, but I'm not sure it makes a big difference.  Changed to beta.